packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -319,6 +310,114 @@
             }
         }
     }
+    &--turbos {
+        background-color: var(--general-section-1);
+        @include mobile {
+            &-open-positions {
+                display: grid;
+                grid-template-columns: 1fr 1fr 1fr;
+                grid-template-areas:
+                    'stake current-stake barrier-level'
+                    'profit-loss limit-order-info';
+                grid-gap: 0.8rem 0.4rem;
+                flex: 1;
+                padding: 0.4rem 0;
+                .dc-contract-card {
+                    &__stake {
+                        grid-column: 1/3;
+                        grid-row: 1/2;
+                        padding: 0.8rem 0;
+                    }
+                    &__current-price {
+                        grid-column: 2/3;
+                        grid-row: 1/2;
+                        padding: 0.8rem 0;
+                    }
+                    &__barrier-level {
+                        grid-column: 3/3;
+                        grid-row: 1/2;
+                        padding: 0.8rem 0;
+                    }
+                    &__buy-price {
+                        grid-column: 1/3;
+                        grid-row: 2/2;
+                        padding: 0.8rem 0;
+                        &--positive {
+                            color: var(--text-loss-success);
+                            &:before {
+                                color: inherit;
+                            }
+                        }
+                        &--negative {
+                            color: var(--text-loss-danger);
+                            &:before {
+                                color: inherit;
+                            }
+                        }
+                    }
+
+                    &__limit-order-info {
+                        grid-row: 2/2;
+                        grid-column: 2/3;
+                        position: relative;
+                        top: 1.3rem;
+                    }
+                    &-dialog-toggle {
+                        width: 100%;
+                        height: calc(100% - 0.9rem);
+                        border: 0.1rem solid #d6dadb;
+                    }
+                    &-item__footer {
+                        background-color: var(--general-section-1);
+                    }
+                    &-item__body {
+                        padding-left: 0;
+                    }
+                }
+            }
+            &-recent-positions {
```

Comment:
@hamza-deriv i think @kate is working on this card for recent-positions, please ask her about it

Reply:
this is the Css which was used in the recent position and disturbing the multiplier css
Thats why i separated it for recent position.